mix.exs

```diff
@@ -51,7 +51,7 @@ defmodule Rajska.MixProject do
     [
       {:ex_doc, "~> 0.19", only: :dev, runtime: false},
       {:credo, "~> 1.5.0", only: [:dev, :test], runtime: false},
-      {:absinthe, "~> 1.4.0 or ~> 1.5.4"},
+      {:absinthe, "~> 1.4.0 or ~> 1.5.4 or ~> 1.6.4"},
```

Comment:
This means rajska doesn't work with versions between 1.6.0 and 1.6.3. Is this true?

If not, we should use `~> 1.6.0`